src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -320,6 +320,18 @@ public sealed partial class JsonIgnoreAttribute : System.Text.Json.Serialization
     {
         public JsonIgnoreAttribute() { }
     }
+    [System.AttributeUsageAttribute(System.AttributeTargets.Property, AllowMultiple=false)]
+    public sealed partial class JsonNameAttribute : System.Text.Json.Serialization.JsonAttribute
+    {
+        public JsonNameAttribute(string name) { }
```

Comment:
Rename argument from name -> propertyName. Same with `ConvertName(name)`

Reply:
Wouldn't apply to ConvertName() based on other discussion.